### ISSUE TITLE
Tooltip with list: accessible name of button is not announced in Edge + Narrator

### ISSUE THREAD

**Issue by jlukosky** [microsoft/fluentui#10272]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

- Microsoft EDGE
- Windows 10 Enterprise 64-bit
- Narrator 

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

Using Edge + Narrator moving focus to the Tooltip with list's example button results in the button's accessible name not being read.

Using Narrator in Edge: 

1. Tab to Tooltip with list section: Hover Over Me button

### Please provide a reproduction of the issue in a codepen:

Not in CodePen, example is live on [Fabric Tooltip Control page](https://developer.microsoft.com/en-us/fabric#/controls/web/tooltip)

**Using Narrator in Edge:** 
1. Move focus to the Hover Over Me button in the "Tooltip with list" section

#### Actual behavior:

Narrator reads :  "One One Two Two Button"
#### Expected behavior:

Narrator should read accessible name and then the description shown in the tooltip: "Hover Over Me One One Two Two Button".
